ros2_control_demo_bringup/config/rrbot_gazebo_joint_trajectory_controller.yaml

```diff
@@ -0,0 +1,31 @@
+controller_manager:
```

Comment:
When we are already working on this example. Would it make sense to add a forwarding controller too? So one can switch controllers is needed? What do you think?

Reply:
Yes that sounds good. I will do that! I think that it will make the difference between the two controllers clearer for the new learners.

Just a question for my understanding: If the user switches to the forwarding controller then there will be snapping movement in between the 4 goal positions right? Since it will forward the commanded position to the joint directly without any control loop. And the joint_traj_controller has the control loop and it keeps the error between commanded and current state so it is a more smoother motion. Do I understand it right?

Reply:
Yes, this is mostly the difference. Actually, difference in trajectory interpolation vs. directly sending commands.